src/main/java/org/owasp/esapi/Validator.java

```diff
@@ -694,6 +694,8 @@
 	 * the kind of regex required for subsequent validation to mitigate regex-based 
 	 * DoS attacks.  
 	 * 
+	 * @see <a href="https://www.ietf.org/rfc/rfc3986.txt">https://www.ietf.org/rfc/rfc3986.txt</a>
```

Comment:
For the label part that's displayed rather than just repeating the URL, I would suggest using something like "RFC 3986 - Uniform Resource Identifier (URI): Generic Syntax".